coderd/coderd.go

```diff
@@ -399,6 +405,13 @@ func New(options *Options) *API {
 		r.Post("/csp/reports", api.logReportCSPViolations)
 
 		r.Get("/buildinfo", buildInfo)
+		r.Route("/config-ssh", func(r chi.Router) {
+			// Require auth for this route to prevent leaking the SSH config.
+			// to non-authenticated users. Also some config settings might
+			// be dependent on the user.
+			r.Use(apiKeyMiddleware)
+			r.Get("/", api.cliSSHConfig)
+		})
```

Comment:
Instead of a new route, what do you think about tossing this under `/deployment`? Maybe `/deployment/ssh`

Reply:
That works :+1: